KIPs/kip-81.md

```diff
@@ -0,0 +1,333 @@
+---
+kip: 81
+title: Implementing the on-chain governance voting method  
+author: Yeriel<yeriel.lee@krustuniverse.com>, Daniel<daniel.cc@krustuniverse.com>, Aidan<aidan.kwon@krustuniverse.com>, Ollie<ollie.j@krustuniverse.com>, Eddie<eddie.kim0@krustuniverse.com>
+discussion-to: https://govforum.klaytn.foundation/t/proposal-implementing-the-on-chain-governance-voting-method/18
+status: Draft
+type: Standards Track 
+category: Core
+created: 2022-09-19
+---
+
+
+## Simple Summary
+Introducing a new governance voting method based on the staking amount and implementation of the Klaytn Square, a governance portal
+ 
+## Abstract
+Klaytn introduces a stake-based governance model that provides voting power to governance participants. Currently, one vote per Governance Council(GC) member was cast. The new method will introduce the voting right that will be exercised based on the staking amount with the maximum cap to prevent an entity from making an arbitrary decision. The voting agenda is determined through discussion on the Klaytn Governance Forum, and the topic will be presented and voted on at the newly launched Governance Portal, Klaytn Square. This voting mechanism aims to provide responsibility and obligation of voting to Governance Councils.  
+
+## Motivation
+The new on-chain voting mechanism discloses GC’s opinion transparently through the public forum and the portal, allowing anyone to view the result. Through Klaytn Governance Forum, non-GC members can also participate in the conversation about governance agenda. This change provides more power and influence to GC members who share the same interest and values as that of Klaytn by staking and locking up more KLAYs. Moreover, this method ultimately promotes decentralization. 
+
+## Specification
+Klaytn Governance Forum allows the general public, developers and governance councils to freely propose and discuss on Klaytn governance items. Once Klaytn Square, the governance portal, opens, the on-chain voting will be executed based on the discussion held in this forum.
```

Comment:
Klaytn Governance Forum and Klaytn Square are different ones, right? How do you want to manage the two sites? Will they be combined into one after the launch of Klaytn Square?

Reply:
The discussion will be first held in the Klaytn Governance Forum. Then, the agenda will be voted on Klaytn Square. They will be managed separately.